install/src/Cabal.hs

```diff
@@ -72,20 +72,17 @@ cabalInstallHls versionNumber args = do
     ++ installMethod
     ++ args
 
-  let minorVerExe = "haskell-language-server-" ++ versionNumber <.> exe
-      majorVerExe = "haskell-language-server-" ++ dropExtension versionNumber <.> exe
+  let verExe = "haskell-language-server-" ++ versionNumber <.> exe
 
   let copyCmd old new = if os == "mingw32"
                         then liftIO $ copyFile old new
                         else command [] "ln" ["-f", old, new]
-  copyCmd (localBin </> "haskell-language-server" <.> exe) (localBin </> minorVerExe)
```

Comment:
Jumm, i think the ghc-x.y.z (`minorVerExe` in the code) is still needed, to have several hls versions and let the wrapper choose the correct one (the `hlsBin` in the wrapper code)
We should remove only the ghc-x.y

Reply:
Ah sorry you have renamed it to verExe, nvm